README.md

```diff
@@ -2,36 +2,219 @@
+
+```python
+import tensorflow as tf
+
+(x_train, y_train), (x_test, y_test) = tf.keras.datasets.mnist.load_data()
+
+mnist_train = tf.data.Dataset.from_tensor_slices((x_train, y_train))
+mnist_test = tf.data.Dataset.from_tensor_slices((x_test, y_test))
+
+BUFFER_SIZE = 10000
+BATCH_SIZE = 64
+
+def scale(image, label):
+    image = tf.cast(image, tf.float32)
+    image /= 255
+    return image, label
+
+train_dataset = mnist_train.map(scale).cache().shuffle(
+    BUFFER_SIZE).batch(BATCH_SIZE)
+eval_dataset = mnist_test.map(scale).batch(BATCH_SIZE)
+
+model = tf.keras.Sequential([
+      tf.keras.layers.Flatten(input_shape=(28, 28)),
+      tf.keras.layers.Dense(512, activation='relu'),
+      tf.keras.layers.Dropout(0.2),
+      tf.keras.layers.Dense(10, activation='softmax')
+    ])
+
+model.compile(loss='sparse_categorical_crossentropy',
+              optimizer=tf.keras.optimizers.Adam(),
+              metrics=['accuracy'])
+
+model.fit(train_dataset, epochs=12)
+```
+
+2. After you have tested this model on your local environment for a few epochs, probably with a small dataset, you can train the model on Google cloud by writing the following simple script `scale_mnist.py`.
+
+```python
+import tensorflow_cloud as tfc
+tfc.run(entry_point='mnist_example.py')
+```
+
+Running this script will automatically apply Tensorflow [Mirrored distribution strategy](https://www.tensorflow.org/api_docs/python/tf/distribute/MirroredStrategy) and train your model at scale on Google Cloud Platform. Please see the [usage guide](#usage-guide) section for detailed instructions on how to use the API.
+
+3. You will see an output similar to the following on your console. The information from the output can be used to track the training job status. 
+
+```console
+usr@desktop$ python scale_mnist.py
+Job submitted successfully.
+Your job ID is:  tf_cloud_train_519ec89c_a876_49a9_b578_4fe300f8865e
+Please access your job logs at the following URL:
+https://console.cloud.google.com/mlengine/jobs/tf_cloud_train_519ec89c_a876_49a9_b578_4fe300f8865e?project=prod-123
+```
+
+## Detailed usage guide
+
+As described in the [high level overview](#high-level-overview) section, the `run` API allows you to train your models at scale on GCP. The [`run`](https://github.com/tensorflow/cloud/blob/master/tensorflow_cloud/run.py#L31) API can be used in four different ways. This is defined by where you are running the API (Terminal vs IPython notebook) and what the `entry_point` parameter value is. `entry_point` is an optional Python script or notebook file path to the file that contains the TensorFlow Keras training code. This is the most important parameter in the API.
+
+
+```python
+run(entry_point=None,
+    requirements_txt=None,
+    distribution_strategy='auto',
+    docker_base_image=None,
+    chief_config='auto',
+    worker_config='auto',
+    worker_count=0,
+    entry_point_args=None,
+    stream_logs=False,
+    docker_image_bucket_name=None,
+    **kwargs)
+```
```

Comment:
At this point, please provide "templates" of how to use `run` to achieve:

- Single V100 training
- 8 V100 training

Reply:
Add a `Cluster configuration and distribution strategy section`